feature-detects/emoji.js

```diff
@@ -9,13 +9,16 @@ Detects support for emoji character sets.
 */
 define(['Modernizr', 'createElement', 'test/canvastext'], function( Modernizr, createElement ) {
   Modernizr.addTest('emoji', function() {
-    if (!Modernizr.canvastext) return false;
+    if (!Modernizr.canvastext || !window.devicePixelRatio) return false;
```

Comment:
we don't want to do that - while I can't think of a browser combination that would fail the second and not the first, it would still be an unnecessary heuristic. Instead, do something like

```